client/lib/promote-post/index.ts

```diff
@@ -62,6 +62,7 @@ declare global {
 					apiRoot: string;
 					headerNonce: string;
 				};
+				jetpackVersion?: string;
 				isV2?: boolean;
```

Comment:
shouldnt we get rid of the line 
isV2 ?

Reply:
Oh, I definitely missed it. Thanks to you, @therocket-gr, it was removed in e5b90879c6d6f052fced5c0d2f2cc0333a913883.